dog-blog-page3.html

```diff
@@ -0,0 +1,61 @@
+<!--Adopt-a-Dog's blog Page 3 -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt-a-Dog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!--this is for the future style sheet-->
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>ADOPTADOG</h1>
+        <img src="adoptadog-logo.svg" width="500" alt="white dog silhoutte">
+        <h2>Seattle's Premier Dog Adoption Agency</h2>
+        <ing src="cart-icon.svg" width="200" alt="cart icon">
+          <a href="dog-checkout-page4.html">Checkout</a>
```

Comment:
all of the links (and file names) can be simplified to 

- index.html
- dog-checkout.html

etc. The page numbers aren't really needed